source/agora/utils/PrettyPrinter.d

```diff
@@ -351,7 +352,9 @@ private struct BlockHeaderFmt
 { utxo: 0x8c15...70e0, seed: 0xaf43...fceb, cycles: 20, sig: 0x0947...1304 }
 { utxo: 0x9490...85b0, seed: 0xa24b...12bc, cycles: 20, sig: 0x0e45...6634 }
 { utxo: 0xb20d...08eb, seed: 0xa050...2cb4, cycles: 20, sig: 0x052e...6b31 }
-{ utxo: 0xdb39...2d85, seed: 0xdd1b...7bfa, cycles: 20, sig: 0x0e00...4fe2 }]`;
+{ utxo: 0xdb39...2d85, seed: 0xdd1b...7bfa, cycles: 20, sig: 0x0e00...4fe2 }]
+Signature: 0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000,
+Validators: [0]`;
```

Comment:
Does this print bit indices which are set?

Reply:
No but if you use a calculator in programming mode you can enter the value and see the bits which are set. Perhaps it would be better to create an output like [0,1,1,1,0,1,1,1] to show the status of bits set though.

Reply:
It's ok for now, we can improve this later.